doc/manuals/orion-api.md

```diff
@@ -1878,89 +1907,100 @@ The values that `options` parameter can have for this specific request are:
-            "status": "failed",
-            "throttling": 5
-          }
-        ]
+* Successful operation uses 200 OK
+* Errors use a non-2xx and (optionally) an error payload. See subsection on "Error Responses" for
+  more details.
 
+Example Response 200:
+
+```json
+[
+  {
+    "id": "62aa3d3ac734067e6f0d0871",
+    "description": "One subscription to rule them all",
+    "subject": {                    
+      "entities": [
+        {
+          "id": "Bcn_Welt",
+          "type": "Room"
+        }
+      ],
+      "condition": {
+          "attrs": [ "temperature " ],
+          "expression": {
+            "q": "temperature>40"
+          }
+      }
+    },
+    "notification": {
+      "httpCustom": {
+        "url": "http://localhost:1234",
+        "headers": {
+          "X-MyHeader": "foo"
+        },
+        "qs": {
+          "authToken": "bar"
+        }
+      },
+      "attrsFormat": "keyValues",
+      "attrs": ["temperature", "humidity"],
+      "timesSent": 12,
+      "lastNotification": "2015-10-05T16:00:00.00Z",
+      "lastFailure": "2015-10-06T16:00:00.00Z"
+    },
+    "expires": "2025-04-05T14:00:00.00Z",
+    "status": "failed",
+    "throttling": 5
+  }
+]
+```
 
 #### Create Subscription [POST /v2/subscriptions]
 
 Creates a new subscription.
 The subscription is represented by a JSON object as described at the beginning of this section.
 
-Response:
+**Request payload**
+
+Content-Type is `application/json`
+
+```json
+{
+  "description": "One subscription to rule them all",
+  "subject": {
+    "entities": [
+      {
+        "idPattern": ".*",
+        "type": "Room"
+      }
+    ],
+    "condition": {
+      "attrs": [ "temperature" ],
+      "expression": {
+        "q": "temperature>40"
+      }
+    }
+  },
+  "notification": {
+    "http": {
+      "url": "http://localhost:1234"
+    },
+    "attrs": ["temperature", "humidity"]
+  },            
+  "expires": "2025-04-05T14:00:00.00Z",
+  "throttling": 5
+}
+```
+
+**Response**
 
 * Successful operation uses 201 Created
 * Errors use a non-2xx and (optionally) an error payload. See subsection on "Error Responses" for
   more details.
 
-+ Request (application/json)
-
-        {
-          "description": "One subscription to rule them all",
-          "subject": {
-            "entities": [
-              {
-                "idPattern": ".*",
-                "type": "Room"
-              }
-            ],
-            "condition": {
-              "attrs": [ "temperature" ],
-              "expression": {
-                "q": "temperature>40"
-              }
-            }
-          },
-          "notification": {
-            "http": {
-              "url": "http://localhost:1234"
-            },
-            "attrs": ["temperature", "humidity"]
-          },            
-          "expires": "2025-04-05T14:00:00.00Z",
-          "throttling": 5
-        }
-
-+ Response 201
+Example Response 201:
 
     + Headers
 
```

Comment:
How this part is rendered is a bit weird

Same as https://github.com/telefonicaid/fiware-orion/pull/4156/files#r911902208